engine/runtime/src/main/scala/org/enso/compiler/context/SuggestionBuilder.scala

```diff
@@ -147,13 +148,14 @@ final class SuggestionBuilder[A: IndexedSource](val source: A) {
     val (methodArgs, returnTypeDef) =
       buildMethodArguments(args, typeSig, selfType)
     Suggestion.Method(
-      externalId    = externalId,
-      module        = module.toString,
-      name          = name.name,
-      arguments     = methodArgs,
-      selfType      = selfType.toString,
-      returnType    = buildReturnType(returnTypeDef),
-      documentation = doc
+      externalId        = externalId,
+      module            = module.toString,
+      name              = name.name,
+      arguments         = methodArgs,
+      selfType          = selfType.toString,
+      returnType        = buildReturnType(returnTypeDef),
+      documentation     = doc,
+      documentationHtml = doc.map(DocsGenerator.runOnPureDoc)
```

Comment:
If the generator function can potentially throw exceptions, they should be handled here.

Reply:
It doesn't really throw much

Reply:
Ok, I take it as "it doesn't throw"